server/graphql.js

```diff
@@ -111,13 +83,15 @@ const getTypedefs = (pathways) => {
     return typeDefs.join('\n');
 }
 
+// Resolvers for GraphQL
 const getResolvers = (config, pathways) => {
     const resolverFunctions = {};
     for (const [name, pathway] of Object.entries(pathways)) {
         if (pathway.disabled) continue; //skip disabled pathways
         resolverFunctions[name] = (parent, args, contextValue, info) => {
             // add shared state to contextValue
             contextValue.pathway = pathway;
+            contextValue.config = config;
```

Comment:
i think this usage of changing contextValue in resolver might be against the best practices but works

Reply:
Yeah it definitely can overwrite pathway and config unexpectedly, but in this case we always want to control what those are anyway so I'll allow it.